### PR TITLE
Add CSS Question on font faces

### PR DESCRIPTION
**Pull request by khairalanam**
## Summary of changes

This PR adds a question to the CSS quiz on font faces.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1672,12 +1672,14 @@ const pythonQuiz = [
   {
     Question:
       "In Python, which of the following dunder methods defines the behavior of the addition operator + for objects of a class?",
+
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1675]: this space doesn't need to be here
please remove

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1672,12 +1672,14 @@ const pythonQuiz = [
...
     Answer: "__add__",
     Distractor1: "__len__",
     Distractor2: "__sub__",
     Distractor3: "__mul__",
     Explanation:
       "Dunder methods (Double underscore) are special methods reserved in Python which affect's the behavior of the object and the interaction of the object with other objects.",
+
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1682]: this space doesn't need to be here
please remove

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jdwilkin4** [comment]
Also as a sidenote, it is best in open source to create branches when submitting pull requests instead of trying to do everything from main.

It will help you avoid stuff like this

Screenshot 2023-10-29 at 10 44 40 AM


Here is an article going more in depth
https://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/

Something to keep in mind about using branches for future PR's 
https://www.freecodecamp.org/news/how-to-create-a-local-git-branch/

File: src/data/python-quiz.ts (hunk 1 above)
Comment (khairalanam) [reply src/data/python-quiz.ts:1675]: Will change it.

File: src/data/python-quiz.ts (hunk 2 above)
Comment (khairalanam) [reply src/data/python-quiz.ts:1682]: Will change it too.

**Comment by khairalanam** [comment]
Thanks for the sidenote! I will be wary of it the next time I send a PR.

**Comment by khairalanam** [comment]
I have implemented the changes. I will send the future PRs from a different branch.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by khairalanam** [comment]
Thank you so much!

**Pull request merged**